cypress/e2e/seededFlows/dashboardFlows/hiddenTags.spec.js

```diff
@@ -0,0 +1,49 @@
+describe('Dashboard: Hidden Tags', () => {
+  beforeEach(() => {
+    cy.testSetup();
+    cy.fixture('users/adminUser.json').as('user');
+
+    cy.get('@user').then((user) => {
+      cy.loginAndVisit(user, '/dashboard/hidden_tags').then(() => {
+        cy.findByRole('heading', { name: 'Dashboard » Hidden tags' });
+      });
+    });
+  });
+
+  it('shows the correct number of tags on the page', () => {
+    cy.get('.dashboard__tag__container').should('have.length', 5);
+  });
+
+  it('shows the appropriate buttons on the card', () => {
+    cy.findByRole('button', { name: 'Unhide tag: tag5' });
+  });
+
+  it('unhides a tag', () => {
+    cy.get('.dashboard__tag__container').should('have.length', 5);
+
+    cy.intercept('/follows').as('followsRequest');
+    cy.findByRole('button', { name: 'Unhide tag: tag5' }).as(
+      'unhideButton',
+    );
+
+    cy.get('@unhideButton').click();
+    cy.wait('@followsRequest');
+
+    // it removes the item from the 'Hidden tags' page
+    cy.get('.dashboard__tag__container').should('have.length', 4);
+    cy.findByRole('button', { name: 'Unhide tag: tag5' }).should(
+      'not.exist',
+    );
+
+    // it decreases the count from the 'Hidden tags' nav item
+    cy.get('.js-hidden-tags-link .c-indicator').as('hiddenTagsCount');
+    cy.get('@hiddenTagsCount').should('contain', '4');
+
+    // it decreases the count from the 'Following tags' nav item
+    cy.get('.js-following-tags-link .c-indicator').as('followingTagsCount');
+    cy.get('@followingTagsCount').should('contain', '6');
+  });
+
+
+  // TODO: add a test for the pagination
```

Comment:
You'll notice this TODO item, currently our pagination and much of the behaviour handled by `initScrolling.js` its not being tested. However, it should be but its a big lift and would increase the scope/effort of this PR. I've made a note of it and will continue chatting to the team about it as its something we will need to tackle before we do the migration.

Reply:
Optional suggestion for this TODO.
File an issue first explain what needs to be done.
Update this comment to mention the issue number too.
Something like this: https://github.com/forem/forem/blob/85b0a18880ab91337299feba51184a7ee08f12aa/app/javascript/admin/controllers/reaction_controller.js#L35

Advantage of this is that if this TODO ends of taking time and we don't prioritise it right now then in future if someone revisits this code and sees the TODO, they can easily visit the issue and get better details. Plus, if someone visits the issue directly, they can also understand whats happening and can work accordingly.

This is completely optional, I followed this on one of my issues.